From 5e1df4b9bb7b80c8ff901785c60f725ce000f24c Mon Sep 17 00:00:00 2001
From: HeYue <yue.he@intel.com>
Date: Wed, 14 Jun 2023 15:33:30 +0800
Subject: [PATCH] Revert "ui/console: fix qemu_console_resize() regression"

This reverts commit 88738ea40bee4c2cf9aae05edd2ec87e0cbeaf36.
It causes SRIOV boot fail. It will cause display corrupt
when changing screen color depth since 7.0.

Signed-off-by: HeYue <yue.he@intel.com>
---
 ui/console.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/ui/console.c b/ui/console.c
index 646202214..29811ed86 100644
--- a/ui/console.c
+++ b/ui/console.c
@@ -2575,13 +2575,11 @@ static void vc_chr_open(Chardev *chr,
 
 void qemu_console_resize(QemuConsole *s, int width, int height)
 {
-    DisplaySurface *surface = qemu_console_surface(s);
+    DisplaySurface *surface;
 
     assert(s->console_type == GRAPHIC_CONSOLE);
 
-    if ((s->scanout.kind != SCANOUT_SURFACE ||
-         (surface && surface->flags & QEMU_ALLOCATED_FLAG)) &&
-        qemu_console_get_width(s, -1) == width &&
+    if (qemu_console_get_width(s, -1) == width &&
         qemu_console_get_height(s, -1) == height) {
         return;
     }
-- 
2.25.1

